### ISSUE TITLE
Mobility is Fucked:  Everyone has Tenacity Softcrit Edition; T-Posing to Assert Dominance in a Berserker Frenzy

### ISSUE THREAD

**Issue by necromanceranne** [tgstation/tgstation#54326]
**Hitting Soft Critical (not stamina critical) doesn't remove handuse**
If you're in softcrit (100-179 damage), you still have full hand use. You're floored, but can still attack. 

**If you buckle yourself to something while in softcrit, you stand up and get all the above benefits while walking but not**
You are able to pass through people as though you're crawling, but you have full hand use. You're immune to stamina damage, and you're also immune to stuns and any form of immobilization once you enter this what we'll call the 'T-Pose Berserk' state.

**This means any ridden vehicles make you able to move at the vehicles speed to no penalty and hulk like invulnerabilities**
Basically, you're kind of nearly invincibility short of someone doing 80 more damage to you. Since we have some chems that help you prevent dying further while in crit, like epinephrine, so long as you have that in you, you're probably not going down.

**Comment by TiviPlus** [comment]
@Rohesie

**Comment by Mey-Ha-Zah** [comment]
I don't know if this an issue or genius.

Either way I say leave it in.

**Comment by necromanceranne** [comment]
Someone yelled at me about the spelling.

**Comment by ATH1909** [comment]
> I don't know if this an issue or genius.
> 
> Either way I say leave it in.

You sure about that, especially with the existence of memento mori?
